### PR TITLE
Added Polar option for get_jacobian()

### PR DESCRIPTION
**Pull request by BlueDrgonDude64**

### CODE REVIEW

File: esa/saw.py [hunk 1]
```diff
@@ -791,7 +791,10 @@ def get_jacobian(self, full=False):
         jidfile = tempfile.NamedTemporaryFile(mode='w', delete=False)
         jidfile_path = Path(jidfile.name).as_posix()
         jidfile.close()
-        cmd = f'SaveJacobian("{jacfile_path}","{jidfile_path}",M,R);'
+        if Polar:
```
Comment (mzy2240) [inline esa/saw.py:794]: Then you can pass the `JacForm` directly to the aux command, which also makes it easier to test

Comment (mzy2240) [reply esa/saw.py:794]: sounds good. have you pushed?

Comment (BlueDrgonDude64) [reply esa/saw.py:794]: Thought I did, but looks like it didn't go through. I'll post a comment again when I push it

Comment (BlueDrgonDude64) [reply esa/saw.py]: I updated the file again, should show now.

File: esa/saw.py [hunk 2]
```diff
@@ -791,9 +791,10 @@ def get_jacobian(self, full=False, Polar=False):
         jidfile = tempfile.NamedTemporaryFile(mode='w', delete=False)
         jidfile_path = Path(jidfile.name).as_posix()
         jidfile.close()
-        if Polar:
-            cmd = f'SaveJacobian("{jacfile_path}","{jidfile_path}",M,P);'
+        if (JacForm == "R") | (JacForm == "P") | (JacForm == "DC")::
```
Comment (mzy2240) [inline esa/saw.py:794]: Actually you should add the docstring for the option `JacForm`, and then you dont need to add the validation logic in the code, cause everyone suppose to read the documentation before using the package

Comment (BlueDrgonDude64) [reply esa/saw.py]: Added the changes, only concern is if {JacForm} needs to have quotes around it or not "{JacForm}"

Comment (mzy2240) [reply esa/saw.py]: no need to add the quotes

**Pull request merged**